kalite/contentload/tests/unpack_assessment_zip.py

```diff
@@ -0,0 +1,100 @@
+import StringIO
+import json
+import os
+import requests
+import zipfile
+from mock import patch, MagicMock, mock_open
+
+from django.conf import settings
+from django.core.management import call_command
+
+from kalite.testing import KALiteTestCase
+from kalite.contentload.management.commands import unpack_assessment_zip as mod
+
+ASSESSMENT_ZIP_SAMPLE_PATH = os.path.join(
+    os.path.dirname(__file__),
+    "fixtures",
+    "assessment_item_resources.sample.zip"
+)
+
+
+class UnpackAssessmentZipCommandTests(KALiteTestCase):
+
+    @patch.object(requests, "get", autospec=True)
+    def test_command_with_url(self, get_method):
+        url = "http://fakeurl.com/test.zip"
+
+        with open(ASSESSMENT_ZIP_SAMPLE_PATH) as f:
+            zip_raw_data = f.read()
```

Comment:
Beware in case we are dealing with bigger video files. Incidentally, I just merged this in django-dbbackup... it's about reading in chunks: https://bitbucket.org/mjs7231/django-dbbackup/commits/c7e0588c3cc5acd3a96bf7708871edb025b859bc